webook/calendar_buddy/contexts/fullcalendar/standard_library.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime
+from .models import EventDisplay
+
+
+def get_base_event_standard() -> dict:
+    """
+        Get the base event standard.
+        This covers the entire fullcalendar event model
+        
+        :return: Returns a dict, containing the base event standards
+        :rtype: dict
+    """
+    event_standard = dict()
+    event_standard["groupId"] = ""
+    event_standard["allDay"] = False
+    event_standard["start"] = datetime.now()
+    event_standard["end"] = datetime.now()
+    event_standard["startStr"] = ""
+    event_standard["endStr"] = ""
+    event_standard["url"] = ""
+    event_standard["classNames"] = []
+    event_standard["editable"] = False
+    event_standard["startEditable"] = False
+    event_standard["durationEditable"] = False
+    event_standard["resourceEditable"] = False
+    event_standard["display"] = EventDisplay.AUTO
+    event_standard["overlap"] = None
+    event_standard["constraint"] = None
+    event_standard["backgroundColor"] = ""
+    event_standard["borderColor"] = ""
+    event_standard["textColor"] = ""
+    event_standard["extendedProps"] = []
+    event_standard["source"] = None
+    return event_standard
```

Comment:
you could just make the dict literally at once instead of assigning keys to values. 


```python 
    return {
        "groupId": "",
        "allDay": False,
        ...: ...,
        "source": None,
    }
    ```